### PR TITLE
Spread Navigation props

### PR DESCRIPTION
**Pull request by Bopchy**
Resolves #NUMBER

**Overall change:** _Spread Navigation props that are not explicitly used in Psammead._

**Code changes:**

- _Spread Navigation props._
- _Update README._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by j-pendlebury — APPROVED** [review]

File: packages/components/psammead-navigation/README.md [hunk 1]
```diff
@@ -206,6 +205,7 @@ import { latin } from '@bbc/gel-foundations/scripts';
   dir={dir}
 />
 ```
+Note that in order for the `AmpMenuButton` toggling to work correctly, an `id` should be added to the `Navigation` component in Psammead. This `id` can be passed in as a prop to the component.
```
Comment (OlgaLyubin) [inline packages/components/psammead-navigation/README.md]: Sorry if I am misunderstanding, but did you mean in Simorgh not Psammead?

Comment (Bopchy) [reply packages/components/psammead-navigation/README.md]: I meant Psammead. I am referring to the addition of an `id` to the Navigation help with the toggling of the classname. 

I can rephrase this to try and make it more clear that it's the class toggling functionality that won't work correctly

Comment (Bopchy) [reply packages/components/psammead-navigation/README.md]: Or maybe I should remove the comment ? As the addition of the id was more of a fix

Comment (OlgaLyubin) [reply packages/components/psammead-navigation/README.md]: No, I think that the comment should be there. I just wanted to clarify about the id because we are now spreading props in the Navigation. All good :) Thank you

Comment (Bopchy) [inline packages/components/psammead-navigation/README.md]:
```suggestion
Note that in order for the `AmpMenuButton` toggling to work correctly, an `id` should be added to the `Navigation` component. This `id` can be passed in as a prop to the component.
```

Comment (Bopchy) [reply packages/components/psammead-navigation/README.md]: Made this(https://github.com/bbc/psammead/pull/2802#discussion_r356530924) change for it to be more clear

**Review by tochwill — APPROVED** [review]
